Rakefile

```diff
@@ -270,6 +270,7 @@ TEST_METADATA = {
 }.freeze
 # rubocop:enable Layout/HashAlignment
 
+desc 'Run test with its coresponding dependencies, example: `bundle exec rake test[spec:redis]`'
```

Comment:
Depending on the shell call `test[spec:redis]` might break.

In Zsh:

```
bundle exec rake test[spec:redis]
zsh: no matches found: test[spec:redis]
```

To fix that we can wrap the brackets with singles quotes 

`bundle exec rake test'[spec:redis]'` I thin is good to update the example to avoid that error from happening